src/odemis/acq/test/stream_test.py

```diff
@@ -4498,7 +4498,7 @@ def test_spectrum_1d(self):
                          spec.metadata[model.MD_PIXEL_SIZE][0])
 
         # compare to doing it manually, by cutting the band at 3
-        sp1d_raw_ex = spec[:, 0, 0, 65:6:-1, 3]
+        sp1d_raw_ex = spec[:, 0, 0, 7:66, 3]
```

Comment:
What do these numbers mean? It seems a bit arbitrary maybe a comment explaining them would be nice?
Or is this just a test set which did work quite good?

Reply:
It's to "manually" do the same thing as done above, using the "fancy" stream interface `.selected_line`.
I've now added some comments to clarify it.